tests/test_pytest_kiwitcms.py

```diff
@@ -76,3 +76,27 @@ def test_empty_variable(pytestconfig):
     result = testdir.runpytest("--kiwitcms", "-v")
     result.stderr.fnmatch_lines(["Exit: Option api_url is empty"])
     assert result.ret == 1
+
+
+def test_kiwitcms_pytest(testdir):
```

Comment:
there are other test functions in this file which are using the `config_test` fixture which contains the URL `http://example.com` which is a real URL serving HTTP but nothing to do with Kiwi TCMS. Can that also be a source of your issues ?

Reply:
Not sure, will try this out as well.

Reply:
I don't think this is the issue